src/seq_mv/csr_matvec_device.c

```diff
@@ -281,6 +393,7 @@ hypre_CSRMatrixMatvecCusparse( HYPRE_Int        trans,
     * they are not present in SpMV interface.
     */
    hypre_CSRMatrixMatvecCusparseNewAPI(trans, alpha, A, x, beta, y, offset);
+
```

Comment:
Do you have multivector support for the old  cusparse API?

Reply:
I don't but I can add that in another PR for completeness

Reply:
Yes, this will be nice. We are still by default using CUDA 10 on lassen now.